jetson/nucleo_bridge/LCMHandler.cpp

```diff
@@ -45,23 +45,20 @@ void LCMHandler::handle_outgoing()
     // This is used as a heart beat (to make sure that nucleos do not reset, but honestly that's too difficult)
     if (NOW - last_heartbeat_output_time > heartbeat_dead_time)
     {
+        internal_object->refresh_ra_quad_angles();
         internal_object->publish_ra_pos_data();
+        internal_object->refresh_sa_quad_angles();
         internal_object->publish_sa_pos_data();
     }
 
 
-    // TODO - UNCOMMENT AS SOON AS JOINT B LIMIT SWITCH / CALIBRATION IS READY
-    /*
     std::chrono::duration calib_data_output_dead_time = std::chrono::milliseconds(1000);
```

Comment:
why is the heartbeat 1 second now? nucleo timoeut is 457 ms

Reply:
i think my comment was misleading, it said it was a heartbeat it's rlly not. I just don't wanna spam calibrated messages so i make it refresh+publish every second instead of every 120 ms

Reply:
why is the dead time 1000 ms though?